autotst/conformer/systematic.py

```diff
@@ -224,23 +178,59 @@ def opt_conf(i, rmsd_cutoff):
             except RuntimeError:
                 logging.info("Optimization failed...we will use the unconverged geometry")
                 pass
-
+        conf_copy = conformer.copy()
         conformer.update_coords_from("ase")  
-        energy = get_energy(conformer)
+        energy = conformer.ase_molecule.get_potential_energy()
         conformer.energy = energy
-        if len(return_dict)>0:
-            conformer_copy = conformer.copy()
-            for index,post in return_dict.items():
-                conf_copy = conformer.copy()
-                conf_copy.ase_molecule.positions = post
-                conf_copy.update_coords_from("ase")
-                rmsd = rdkit.Chem.rdMolAlign.GetBestRMS(conformer_copy.rdkit_molecule,conf_copy.rdkit_molecule)
-                if rmsd <= rmsd_cutoff:
-                    return True
-        if str(i) != 'ref':
-            return_dict[i] = conformer.ase_molecule.get_positions()
-        return True
+        
+        return conformer
+def systematic_search(conformer,
+                      delta=float(120),
+                      energy_cutoff = 10.0, #kcal/mol
```

Comment:
Always been a lil unsure about why we chose 10.0 kcal/mol as our cut off... @davidfarinajr? Probably not needed for this PR, but we should figure out a way to justify why we use 10 for future uses.

Reply:
I think 10.0 kcal/mol is a conservative cutoff.  It's what I have been using because I dont have high confidence in DFTB energies, but maybe we should reduce the default to 5?